src/base/bittorrent/torrenthandle.cpp

```diff
@@ -1405,6 +1415,16 @@ void TorrentHandle::toggleFirstLastPiecePriority()
     setFirstLastPiecePriority(!hasFirstLastPiecePriority());
 }
 
+void TorrentHandle::setPauseOnCompletion(bool b)
+{
+    m_pauseOnCompletion = b;
+}
+
+void TorrentHandle::togglePauseOnCompletion()
```

Comment:
This method is too trivial to overcharge **TorrentHandle** interface.

Reply:
It's as trivial as `void TorrentHandle::toggleFirstLastPiecePriority()`

Reply:
Then we should remove it as well.